agent/setup.py

```diff
@@ -0,0 +1,37 @@
+from os import path
+from setuptools import setup, find_packages
+
+import balrogagent
+
+here = path.abspath(path.dirname(__file__))
+# We're using a pip8 style requirements file, which allows us to embed hashes
+# of the packages in it. However, setuptools doesn't support parsing this type
+# of file, so we need to strip those out before passing the requirements along
+# to it.
+with open(path.join(here, "requirements.txt")) as f:
+    requirements = []
+    for line in f:
+        if "hash" not in line:
```

Comment:
Hmm,  this wouldn't let you list anything with "hash" substring. Maybe use something like this instead:
```python
requirements = [line.split()[0] for line in f.readlines() if not line.strip().startswith("#")]
```